src/update-feeds/dapi-data-registry.ts

```diff
@@ -24,6 +26,31 @@ export const decodeDapisCountResponse = (dapiDataRegistry: DapiDataRegistry, dap
 
 export type DapisCountResponse = ReturnType<typeof decodeDapisCountResponse>;
 
+export const decodeDataFeed = (dataFeed: string): DecodedDataFeed => {
+  if (dataFeed.length === 130) {
+    // (64 * 2) - 2
```

Comment:
```suggestion
    // (64 * 2) + 2
```

and it would be nice to explain why *2 and also +2